flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyMessageClientSideSerializationTest.java

```diff
@@ -81,11 +95,47 @@
 		});
 	}
 
-	public NettyMessageSerializationTest(boolean testReadOnlyBuffer, boolean testCompressedBuffer) {
+	public NettyMessageClientSideSerializationTest(boolean testReadOnlyBuffer, boolean testCompressedBuffer) {
 		this.testReadOnlyBuffer = testReadOnlyBuffer;
 		this.testCompressedBuffer = testCompressedBuffer;
 	}
 
+	@Before
+	public void setup() throws IOException, InterruptedException {
+		networkBufferPool = new NetworkBufferPool(10, 1024, 2);
+		BufferPool bufferPool = networkBufferPool.createBufferPool(8, 8);
+
+		inputGate = new SingleInputGateBuilder()
+			.setNumberOfChannels(1)
+			.setBufferPoolFactory(bufferPool)
+			.build();
+		inputChannel = createRemoteInputChannel(
+			inputGate,
+			mock(PartitionRequestClient.class),
+			networkBufferPool);
+		inputGate.assignExclusiveSegments();
+		inputChannel.requestSubpartition(0);
+
+		CreditBasedPartitionRequestClientHandler handler = new CreditBasedPartitionRequestClientHandler();
```

Comment:
CreditBasedPartitionRequestClientHandler -> NetworkClientHandler

Reply:
Fixed according to the comment.